rakelib/test.rake

```diff
@@ -87,6 +87,28 @@ namespace "test" do
     task.reenable
   end
 
+  task "integration" => ["setup"] do
```

Comment:
If you depends on bootstrap task, you will run into `bundler` issues, trust me I didn't learn my lesson the first time and tried to make it work. :cry: 

If you look at the current workflow of the tests they don't bootstrap automatically.

rake bootstrap https://github.com/elastic/logstash/blob/master/ci/ci_setup.sh#L17
rake test:install-core https://github.com/elastic/logstash/blob/master/ci/ci_test.sh#L12
rake test:core https://github.com/elastic/logstash/blob/master/ci/ci_test.sh#L13

If you look at the `test:core` it depends on setup too.
https://github.com/ph/logstash/blob/fix/integration-test/rakelib/test.rake#L23-L25